src/test/java/racing/CarTest.java

```diff
@@ -1,46 +1,52 @@
 package racing;
 
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import org.junit.jupiter.params.provider.CsvSource;
+import racing.strategy.MoveStrategy;
 
-import static org.assertj.core.api.Assertions.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 class CarTest {
 
-    Car car;
+    @Test
+    void play() {
+        MoveStrategy moveStrategy = () -> true;
```

Comment:
다른 테스트에서도 반복해서 사용하니 멤버 변수로 두면 어떨까요?
변수명도 의도를 쉽게 파악할 수 있는 이름이면 좋을 것 같아요
```suggestion
        private final MoveStrategy alwaysMove = () -> true;
```